signals.mss

```diff
@@ -1301,7 +1301,8 @@ Format details:
   /****************************/
   /* NL main light signals */
   /****************************/
-  ["feature"="NL"]["main_form"="light"] {
+  ["feature"="NL"]["main_form"="light"],
+  ["feature"="NL"]["main_form"="NL:light"] {
```

Comment:
wrong tagging

Reply:
I agree this is a tagging mistake. I will fix the tagging on some of the railway lines in the Netherlands.

Reply:
Done in mass edit https://www.openstreetmap.org/changeset/142925660#map=10/52.2317/5.8090